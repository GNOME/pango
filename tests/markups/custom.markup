<markup>
<span font="Cantarell 20">
  <span font_family="Noto Sans">noto</span>
  <span font_size="30">size</span>
  <span font_style="italic">style</span>
  <span font_weight="738">weight</span>
  <span font_variant="small-caps">variant</span>
  <span font_stretch="ultra-expanded">stretch</span>
  <span color="red">red</span>
  <span fgcolor="#ff0000ff">red</span>
  <span bgcolor="#00aabbcc">not red</span>
  <span segment="word">word</span>
  <span segment="sentence">this is a sentence.</span>
  <span segment="paragraph">this is a paragraph.</span>
  <span size="smaller">smaller</span>
  <span size="larger">larger</span>
  <span size="xx-large">xx</span>
  <span size="200%">double</span>
</span>
</markup>
