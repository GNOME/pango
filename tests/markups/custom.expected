

  noto
  size
  style
  weight
  variant
  stretch
  red
  red
  not red
  word
  this is a sentence.
  this is a paragraph.
  smaller
  larger
  xx
  double




---

range 0 1
range 1 4
1 160 font-desc "Cantarell 20"
range 4 8
1 160 font-desc "Cantarell 20"
4 8 family "Noto Sans"
range 8 11
1 160 font-desc "Cantarell 20"
range 11 15
1 160 font-desc "Cantarell 20"
11 15 size 30
range 15 18
1 160 font-desc "Cantarell 20"
range 18 23
1 160 font-desc "Cantarell 20"
18 23 style italic
range 23 26
1 160 font-desc "Cantarell 20"
range 26 32
1 160 font-desc "Cantarell 20"
26 32 weight 738
range 32 35
1 160 font-desc "Cantarell 20"
range 35 42
1 160 font-desc "Cantarell 20"
35 42 variant small-caps
range 42 45
1 160 font-desc "Cantarell 20"
range 45 52
1 160 font-desc "Cantarell 20"
45 52 stretch ultra-expanded
range 52 55
1 160 font-desc "Cantarell 20"
range 55 58
1 160 font-desc "Cantarell 20"
55 58 foreground #ffff00000000
range 58 61
1 160 font-desc "Cantarell 20"
range 61 64
1 160 font-desc "Cantarell 20"
61 64 foreground #ffff00000000
range 64 67
1 160 font-desc "Cantarell 20"
range 67 74
1 160 font-desc "Cantarell 20"
67 74 background #0000aaaabbbb
67 74 background-alpha 52428
range 74 77
1 160 font-desc "Cantarell 20"
range 77 81
1 160 font-desc "Cantarell 20"
77 81 word true
range 81 84
1 160 font-desc "Cantarell 20"
range 84 103
1 160 font-desc "Cantarell 20"
84 103 sentence true
range 103 106
1 160 font-desc "Cantarell 20"
range 106 126
1 160 font-desc "Cantarell 20"
106 126 sentence true
range 126 129
1 160 font-desc "Cantarell 20"
range 129 136
1 160 font-desc "Cantarell 20"
129 136 size 17066
range 136 139
1 160 font-desc "Cantarell 20"
range 139 145
1 160 font-desc "Cantarell 20"
139 145 size 24576
range 145 148
1 160 font-desc "Cantarell 20"
range 148 150
1 160 font-desc "Cantarell 20"
148 150 scale 1.728000
range 150 153
1 160 font-desc "Cantarell 20"
range 153 159
1 160 font-desc "Cantarell 20"
153 159 scale 2.000000
range 159 160
1 160 font-desc "Cantarell 20"
range 160 2147483647


---

[0:1] (null) Normal
[1:4] (null) Cantarell 20
[4:8] (null) Noto Sans 20
[8:11] (null) Cantarell 20
[11:15] (null) Cantarell 0.029296875
[15:18] (null) Cantarell 20
[18:23] (null) Cantarell Italic 20
[23:26] (null) Cantarell 20
[26:32] (null) Cantarell weight=738 20
[32:35] (null) Cantarell 20
[35:42] (null) Cantarell Small-Caps 20
[42:45] (null) Cantarell 20
[45:52] (null) Cantarell Ultra-Expanded 20
[52:55] (null) Cantarell 20
[55:58] (null) Cantarell 20
[58:61] (null) Cantarell 20
[61:64] (null) Cantarell 20
[64:67] (null) Cantarell 20
[67:74] (null) Cantarell 20
[74:77] (null) Cantarell 20
[77:81] (null) Cantarell 20
[81:84] (null) Cantarell 20
[84:103] (null) Cantarell 20
[103:106] (null) Cantarell 20
[106:126] (null) Cantarell 20
[126:129] (null) Cantarell 20
[129:136] (null) Cantarell 16.666015625
[136:139] (null) Cantarell 20
[139:145] (null) Cantarell 24
[145:148] (null) Cantarell 20
[148:150] (null) Cantarell 34.5595703125
[150:153] (null) Cantarell 20
[153:159] (null) Cantarell 40
[159:160] (null) Cantarell 20
[160:2147483647] (null) Cantarell 20
